src/layers/normalise.jl

```diff
@@ -65,6 +65,27 @@ Specify `rng` to use a custom RNG instead of the default.
 Custom RNGs are only supported on the CPU.
 
 Does nothing to the input once [`Flux.testmode!`](@ref) is `true`.
+
+# Examples
+```jldoctest
+julia> m = Chain(Dense(1 => 1), Dropout(1));
+
+julia> Flux.trainmode!(m);
+
+julia> y = m([1]);
+
+julia> count(i->(i == 0), y) == m[2].p  # number of zeros == 1
+true
+
+julia> m = Chain(Dense(1 => 1), Dropout(0.5));
+
+julia> Flux.trainmode!(m);
+
+julia> y = m([1]);
+
+julia> m[2].p - 0.5 <= count(i->(i == 0), y) <= m[2].p + 0.5  # number of zeros can be 0 or 1
```

Comment:
```suggestion
julia> y = m(ones(1000));

julia> isapprox(count(==(0), y) / length(y), 0.5, atol=0.1)
```
I'm not sure if this tolerance is enough, so you may have to do some tweaking.